fetch/api/resources/stash-take.py

```diff
@@ -4,6 +4,6 @@
 @json_handler
 def main(request, response):
     dir = '/'.join(request.url_parts.path.split('/')[:-1]) + '/'
```

Comment:
I think this should be
```suggestion
    dir = u'/'.join(request.url_parts.path.split(u'/')[:-1]) + u'/'
```

Basically `request.url_parts.path` is unfortunately still a native string (#23362 ), and mixing it with Unicode literals works everywhere (in Python 2 you can mix `unicode` with `str` even though they are different, and in Python 3 they are the same). Yeah I know...